packages/tldraw/src/lib/shapes/note/NoteShapeUtil.tsx

```diff
@@ -209,6 +209,8 @@ export class NoteShapeUtil extends ShapeUtil<TLNoteShape> {
 			this.editor,
 		])
 
+		const darkMode = useValue('dark mode', () => this.editor.user.getIsDarkMode(), [this.editor])
```

Comment:
I don't think it makes sense to show bottom border when using dark mode (we even have a question in this file if using the drop shadow makes sense for dark mode).